src/openklant/components/klantinteracties/models.py

```diff
@@ -0,0 +1,340 @@
+import uuid
+
+from django.core.validators import MinLengthValidator, validate_integer
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from vng_api_common.descriptors import GegevensGroepType
+
+from .constants import SoortPartij
+
+
+class Partij(models.Model):
+    id = models.UUIDField(
+        primary_key=True,
```

Comment:
```suggestion
    uuid = models.UUIDField(
```

Mag overal doorgevoerd worden.